server/functions/__tests__/mocks/api/.nvmrc

```diff
@@ -0,0 +1 @@
+14
```

Comment:
Should mention this in the requirements section of the PR.

Reply:
Added.